src/app/student/map/_components/BoothPopup.tsx

```diff
@@ -0,0 +1,19 @@
+"use client"
+
+import { Popup } from "react-map-gl/maplibre"
+import { Booth } from "../lib/booths"
+
+export function BoothPopup({ booth }: { booth: Booth }) {
+  return (
+    <div>
```

Comment:
Removed

Reply:
Now that I think about it we should maybe just remove the popups completely, the only thing they display atm is the exhibitor name but that is not very useful since we already show detailed exhibitor information in the sidebar when you click on a booth, and not sure what else you could put in the popup. Thoughts?

Reply:
Hi! Me and August discussed this slightly and I think that if we're keeping them then we should add more content to them (kind of like the popup in larm), if we don't want to add more content to them then we can remove the entire popup. I don't think that's a blocking thing for this PR though